docs/user/rstudio/ide/guide/code/execution.qmd

```diff
@@ -2,29 +2,29 @@
 title: "Executing code"
 ---
 
-RStudio supports the direct execution of code from within the source editor (the executed commands are inserted into the console where their output also appears).
+RStudio supports the direct execution of code from within the source editor (the executed commands are inserted into the console where the output also appears).
 
 ## Source files
 
 Source code files such as .R or .py can be executed in several different ways.
 
 ### Executing a Single Line
 
-To execute the line of source code where the cursor currently resides you press the **Ctrl+Enter** key (or use the **Run** toolbar button):
+To execute the line of source code where the cursor currently resides press the <kbd>Ctrl+Enter</kbd> keys (or use the **Run** toolbar button):
 
 ![](images/run-current-line.png){fig-alt="A screenshot of the RStudio source pane and the 'Run' button."}
 
-After executing the line of code, RStudio automatically advances the cursor to the next line. This enables you to single-step through a sequence of lines.
+After executing the line of code, RStudio automatically advances the cursor to the next line. This enables single-stepping through a sequence of lines.
 
 ### Executing Multiple Lines
 
 There are three ways to execute multiple lines from within the editor:
 
--   Select the lines and press the **Ctrl+Enter** key (or use the **Run** toolbar button); or
+-   Select the lines and press the <kbd>Ctrl+Enter</kbd> keys (or use the **Run** toolbar button); or
 
 -   After executing a selection of code, use the **Re-Run Previous Region** command (or its associated toolbar button) to run the same selection again. Note that changes to the selection including additional, removal, and modification of lines will be reflected in this subsequent run of the selection.
```

Comment:
The Re-Run Previous Region button is super cool and I'm bummed that I never noticed it before! I think it's kinda easy to skip over in that section of the toolbar. What do you think about adding a new screenshot with the Run, Re-Run, and Source buttons labeled like in the **Executing Multiple Lines** image below?

Reply:
Done!